mcs/class/Mono.Posix/Mono.Unix/UnixEncoding.cs

```diff
@@ -279,38 +279,26 @@ public override int GetByteCount (String s)
 		return InternalGetBytes (chars, charIndex, charCount, bytes, byteIndex, ref leftOver, true);
 	}
 
-	// Convenience wrappers for "GetBytes".
-	public override int GetBytes (String s, int charIndex, int charCount,
-								 byte[] bytes, int byteIndex)
+	public unsafe override int GetBytes(char* chars, int charCount, byte* bytes, int byteCount)
```

Comment:
To reduce the patch size, could you place this override *after* the existing `GetBytes(String...` overload? I think that will appease the `diff -u` gods.